src/homework26/DataBaseConnection.java

```diff
@@ -0,0 +1,57 @@
+package homework26;
+
+import org.postgresql.util.PSQLException;
+
+import java.sql.*;
+
+public class DataBaseConnection {
+    //метод getConnection() - повертає нове з'єднання з БД
+    //метод close(Connection) - закриває передане з'єднання
+    public Connection getConnection() throws SQLException {
+        Connection conn = DriverManager.getConnection( "jdbc:postgresql://127.0.0.1:5432/hw24", "postgres", "onl1918s");
+        if (conn != null) {
+            System.out.println("Connected to the database!");
+        } else {
+            System.out.println("Failed to make connection!");
+        }
+        return conn;
+    }
+
+    public void close(Connection conn) throws SQLException {
+        conn.close();
+        System.out.println("Connection is closed");
+    }
+
+    //Method for executing queries
+    public void execute(String query) throws SQLException {
```

Comment:
Those two methods (`execute` and `executeId`) shouldn't be here. This class is responsible only for connection. Those methods must be in DAO impl.